src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/AnalysesListingService.java

```diff
@@ -166,7 +166,7 @@ private JobError getFirstJobError(AnalysisSubmission submission) throws Executio
 	 * @param end   {@link Date}
 	 * @return {@link Long} milliseconds
 	 */
-	private Long getDurationInMilliseconds(Date start, Date end) {
```

Comment:
So I have been thinking about this a bit.  Copy out of my branch `analyses-base` re-create the file I made `src/main/java/ca/corefacility/bioinformatics/irida/ria/web/utilities/DateUtilities.java` and use the static method `getDurationInMilliseconds`, that way we should eliminate some conflicts when we merge the two branches.

Reply:
Fixed in 9c4e640d5c452bf1455f9b93a1b1499a418283e6

Reply:
You said this was fixed, but you did not do this?